### DISCUSSION TITLE
about project

### DISCUSSION THREAD

**Discussion by cc6522992** [florisboard/florisboard#2073] [Q&A] [answered]
Is there a florisboard written in compose?
version project?

**Comment by tsiflimagas** [comment]
I'm not sure I understand your question, the phrasing is somewhat rough. I think you mean what versions of FlorisBoard are built with compose? The first stable release built using compose was 0.3.14. Since then, all versions are built using compose, of course.

**Reply by cc6522992** [reply]
Is there a version without compose?

**Reply by lm41** [reply, marked as answer]
No
